tests/21-ct-clean-up.sh

```diff
@@ -73,7 +87,7 @@ function test_reachability(){
   local container="${1}"
   local dst_ip="${2}"
   local http_path="${3}"
-  log "trying to curl http://${dst_ip}:80${http_path} from client container (should work)"
+  log "trying to docker exec -i "${container}" bash -c \"curl -s -o /dev/null -w \"%{http_code}\" --connect-timeout $TIMEOUT -XGET http://${dst_ip}:80${http_path}\" from client container (should work)"
```

Comment:
I think you can remove the quotes before and after `${container}`

Reply:
Done